packages/components/TeamPage.tsx

```diff
@@ -13,38 +11,28 @@ type Props = {
 export default function TeamPage({ members }: Props) {
   return (
     <Section className="py-40">
-      <Grid gap={10}>
-        <Heading fontWeight={600} fontSize={{ base: "3xl", sm: "4xl" }} lineHeight="110%" textAlign="center">
+      <div className="grid gap-10">
+        <h2 className="font-semibold text-4xl sm:font-xl text-center leading-8">
```

Comment:
This is the page's main title, so it must be an `h1` for correct syntax.

```suggestion
        <h1 className="font-semibold text-4xl sm:font-xl text-center leading-8">
```

Reply:
change here [c4898bf](https://github.com/podcodar/webapp/pull/216/commits/c4898bfcda80c889a0429fe4e2b1db38301d82d1)